File: GymRatAPI/GymRat/src/main/java/com/appMoviles/GymRat/entities/Ejercicio.java
package com.appMoviles.GymRat.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "ejercicios")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Ejercicio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, length = 100)
    private String nombre;

    @Column(nullable = false, length = 50)
    private String categoria;

    @Column(nullable = false, length = 300)
    private String descripcion;

    @Column(nullable = false, length = 100)
    private String musculoPrincipal;

    @Column(nullable = false)
    private Integer seriesRecomendadas;

    @Column(nullable = false)
    private Integer repeticionesRecomendadas;

    @Column(nullable = false, length = 30)
    private String nivel;
}

File: GymRat2.0/app/src/main/java/com/appmoviles/gymrat20/clases/Ejercicio.kt
package com.appmoviles.gymrat20.clases

data class Ejercicio(
    val id: Long? = null,
    val nombre: String,
    val categoria: String,
    val descripcion: String,
    val musculoPrincipal: String,
    val seriesRecomendadas: Int,
    val repeticionesRecomendadas: Int,
    val nivel: String
)